frontend/www/js/omegaup/components/arena/CodeView.vue

```diff
@@ -116,6 +116,19 @@ export default class CodeView extends Vue {
   onLanguageChange(newLanguage: string) {
     this.mode = languageModeMap[newLanguage];
   }
+  mounted() {
+    const codeAndLanguage = {
+      code: this.value,
+      language: !this.language ? 'cpp17-gcc' : this.language,
```

Comment:
If you require a default value, you can define the property's default value as follows:
```ts
  @Prop({ default: 'cpp17-gcc' }) language!: string;
```
Afterward, there's no need for the ternary operator in this context. You can simply assign the property's value like this:
```suggestion
      language: this.language,
```